core/src/test/scala/integration/kafka/admin/ListOffsetsIntegrationTest.scala

```diff
@@ -66,7 +66,7 @@ class ListOffsetsIntegrationTest extends KafkaServerTestHarness {
   }
 
   @ParameterizedTest
-  @ValueSource(strings = Array("zk", "kraft"))
+  @ValueSource(strings = Array("kraft"))
   def testListMaxTimestampWithEmptyLog(quorum: String): Unit = {
```

Comment:
Isn't the proper change here to remove the quorum parameter entirely?

Reply:
I'd wondered that and wanted someone to raise the question.

I think it makes sense to remove it.

Reply:
Happy to oblige :)

Reply:
I have a question out to @cmccabe regarding his preference.

Reply:
`TestInfoUtils` uses the parameters to determine if it `isKraft()`, `isZkMigrationTest()`, etc. Usages of those methods would need to be updated too.

Reply:
Yes, I'm sure, but wouldn't AK 4.0 have all of the ZK migration tests removed, and wouldn't all tests be ZK?

Reply:
There are some shenanigans done in the base test classes regarding the `quorum` param. Until we remove the ZK bits from that test infra, I think we actually need the `quorum` param :-/

Reply:
I approve of the use of "shenanigans" to describe this 👍